GooglePlacesAutocomplete.js

```diff
@@ -525,6 +526,7 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
             }
           }
         } else {
+            setListViewDisplayed(false);
```

Comment:
💅  this is indented once too many

```suggestion
          setListViewDisplayed(false);
```

Reply:
😭  my linter doesn't seem to want to play ball in there